zproject/settings.py

```diff
@@ -1030,6 +1038,16 @@ def get_secret(key):
             'handlers': ['file'],
             'propagate': False,
         },
+        'django.server': {
+            'propagate': False,
+            'filters': ['skip_200_and_304'],
+        },
+        'django.template': {
+            'handlers': ['console'],
+            'filters': ['require_debug_true'],
+            'level': 'DEBUG',
+            'propgate': False,
+        },
```

Comment:
`propagate` is mispelled; I'm also not sure why that `django.template` logging configuration is correct; I would think that we actually want `propagate=True`, so that `WARNING` level django.template log items still show up, right?